src/components/wallet/UnlockForm.style.tsx

```diff
@@ -0,0 +1,90 @@
+import AF from 'antd/lib/form'
+import AT from 'antd/lib/typography/Text'
+import styled from 'styled-components'
+import { palette } from 'styled-theme'
+
+import { media } from '../../helpers/styleHelper'
+import BaseButton from '../uielements/button'
+
+export const Form = styled(AF)`
+  display: flex;
+  flex-direction: column;
+
+  ${media.lg`
+    height: 100%;
+  `}
+
+  .ant-form-item {
+    width: 100%;
+    margin: 0;
+  }
+`
+
+export const Header = styled('div')`
+  display: flex;
+  justify-content: center;
+  position: relative;
+
+  .ant-typography {
+    font-weight: 600;
+  }
+`
+
+export const Text = styled(AT)`
+  width: 100%;
+  text-align: center;
+  display: inline-block;
+  margin-bottom: 30px;
+  color: ${palette('text', 1)};
+  text-transform: uppercase;
+`
+
+export const PasswordInput = styled(AF.Item)`
+  width: 100%;
+  max-width: 400px;
+  margin-bottom: 20px !important;
+`
+
+export const Content = styled('div')`
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+  height: 100%;
+  padding: 45px 30px 35px 30px;
+  background-color: ${palette('background', 1)};
+
+  ${media.sm`
+    padding: 90px 60px 70px 60px;
+  `}
+
+  .ant-row {
```

Comment:
Same here: Is there another way to style this instead of using an `antd` class selector?